core/atomic_c11.c

```diff
@@ -330,7 +337,7 @@ void __atomic_exchange_c(size_t size, void *ptr, void *val, void *ret, int memor
  * @return false otherwise
  */
 bool __atomic_compare_exchange_c(size_t len, void *ptr, void *expected,
-    void *desired, bool weak, int success_memorder, int failure_memorder)
+                                 void *desired, bool weak, int success_memorder, int failure_memorder)
```

Comment:
This was delivered by `uncrustify` this way? `int failure_memorder` could (and should) be on the next line.

Reply:
It seems like our uncrustify config doesn't do line wrapping at all. I've pushed a change to make it do it.

It feels quite intrusive... ;)

Reply:
> It feels quite intrusive... ;)

If it formats it this way it's plain wrong though.

Reply:
> If it formats it this way it's plain wrong though.

Hmyeah. I tried limiting uncrustify's line wrap to 100 cols. It prevents this mishap.

Reply:
An alternative is setting ```ls_func_split_full```, which results in this (if a line is longer than the configured limit):

```
-bool __atomic_compare_exchange_c(size_t len, void *ptr, void *expected,
-                                 void *desired, bool weak, int success_memorder, int failure_memorder)
+bool __atomic_compare_exchange_c(size_t len,
+                                 void *ptr,
+                                 void *expected,
+                                 void *desired,
+                                 bool weak,
+                                 int success_memorder,
+                                 int failure_memorder)
```

Reply:
Don't like that style (and only use it if everything else doesn't make sense stylistically)

Reply:
I've split out the 100char config change into another PR: #10873